src/acc/libsmm_acc/libcusmm/generate_parameters.py

```diff
@@ -19,7 +19,7 @@ def main(argv):
 
     # Read existing parameters
     print("GPU version:\n", options.gpu_version)
-    param_fn = "parameters_" + options.gpu_version + ".txt"
+    param_fn = "parameters_" + options.gpu_version + ".json"
```

Comment:
In terms of readability I personally prefer [format-strings](http://pyformat.info) instead:

```python
param_fn = "parameters_{}.json".format(options.gpu_version)
```

... maybe something for next time.

Reply:
ok, I'll keep it in mind for next time